WordPress/Classes/Models/UserSuggestion+Comparable.swift

```diff
@@ -0,0 +1,11 @@
+extension UserSuggestion: Comparable {
+    public static func < (lhs: UserSuggestion, rhs: UserSuggestion) -> Bool {
+        if let leftDisplayName = lhs.displayName, let rightDisplayName = rhs.displayName {
+            return leftDisplayName.localizedCaseInsensitiveCompare(rightDisplayName) == .orderedAscending
+        } else if let leftUsername = lhs.username, let rightUsername = rhs.username {
+            return leftUsername < rightUsername
```

Comment:
Why aren't we using `localizedCaseInsensitiveCompare` here?

Reply:
@hassaanelgarem Quoting @igoriols from pbMoDN-3N0-p2#implementation:

> I’m using `localizedCaseInsensitiveCompare` [to compare displayNames], because the user can have the `displayName` in different languages. The comparison is localizable and case-insensitive.
>
> I’m using `<` to compare usernames because they are very restricted, and the lexicographical comparison should work for them.